app/schema_translation.py

```diff
@@ -18,7 +18,7 @@ def save(self, translation_file_path):
         with open(translation_file_path, 'w+b') as translation_file:
             pofile.write_po(translation_file, self.catalog)
 
-    def translate_message(self, message_to_translate, answer_id=None):
+    def translate_message(self, message_to_translate, answer_id=None, user_context=None):
```

Comment:
This makes much more sense if `user_context` is renamed `message_context`.

Reply:
This was a hangover from the multiple types of context we have and what I'd assumed we were using during extraction. Updated.